src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int TEST_SIZE = 6;
+
+    private static FigureSupplier figureSupplier = new FigureSupplier();
```

Comment:
The variable 'figureSupplier' should not be static. Static methods and variables are generally discouraged. Instead, create an instance of 'FigureSupplier' within the 'main' method.